packages/webview_flutter/example/lib/main.dart

```diff
@@ -53,26 +77,31 @@ class WebViewExample extends StatelessWidget {
           return Container();
         });
   }
+
+  Future<dynamic> _runSampleJSEvaluation(WebViewController controller) async {
```

Comment:
nit: Flutter's naming style would be to capitalize just the first letter even if it's an acronym so `Js` here.